scripts/install.js

```diff
@@ -0,0 +1,28 @@
+// Runs npm ci in each of the libraries
```

Comment:
This vs npm workspaces?

Reply:
Doing that as a separate change, this is preserving existing behavior

Reply:
Hm, seems like karma webpack isn't happy, can't find modules. I guess it doesn't do full node module resolution